test/support/utils.rb

```diff
@@ -18,11 +18,14 @@ def assert_line_num(expected)
 
     def assert_line_text(expected)
       @queue.push(Proc.new {
-        assert_block do
-          @internal_info['backlog'].each do |line|
-            line.include? expected
+        result = false
+        @internal_info['backlog'].each do |line|
+          if line.include? expected
```

Comment:
can you use `match?` instead of `include?` so it can also take regexp as expectation?

Reply:
https://github.com/ruby/debug/pull/71/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R23